environment.server.yml

```diff
@@ -1,3 +1,4 @@
 name: deepforge
 dependencies:
   - python=3.7
+  - python-language-server
```

Comment:
Is this still needed? Aren't we running the language servers using the docker image?

Reply:
This is needed because in the [Dockerfile.langservers](https://github.com/deepforge-dev/deepforge/pull/1910/files#diff-3892821bf585a3546fec594edbe738bfR13-R15) we are using the file `src/plugins/GenerateJob/templates/environment.worker.yml` for updating the base environment with worker dependencies.

If you have a better alternative, I am not opposed to changing it.

Reply:
I think we can remove this here and install language server in the docker image directly.

Reply:
Shouldn't this have been removed?